response_operations_ui/templates/admin/manage-account.html

```diff
@@ -0,0 +1,164 @@
+                            })
+                        }}
+                        </td>
+                        <td class="ons-table__cell">
+                        {{
+                            onsCheckbox({
+                                "id": "respondents_delete",
+                                "name": "respondents_delete",
+                                "label": {
+                                    "text": "Delete respondents"
+                                },
+                                "hideLabel": true,
+                                "value": "y",
+                                "checked": permissions["respondents.delete"]
+                            })
+                        }}
+                        </td>
+                    </tr>
+                    <tr class="ons-table__row">
+                        <td class="ons-table__cell">
+                            Messages
+                        </td>
+                        <td class="ons-table__cell">
+                        {{
+                            onsCheckbox({
+                                "id": "messages_edit",
+                                "name": "messages_edit",
+                                "label": {
+                                    "text": "Edit messages"
+                                },
+                                "hideLabel": true,
+                                "value": "y",
+                                "checked": permissions["messages.edit"]
+                            })
+                        }}
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+
+          <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-u-mt-m ons-u-mb-m">
+            <div class="ons-grid__col ons-col-3@m">
+            <strong>Manage user accounts</strong>
+            {{
+                onsCheckbox({
+                    "id": "users_admin",
+                    "name": "users_admin",
+                    "label": {
+                        "text": "Manage user accounts"
+                    },
+                    "value": "y",
+                    "checked": permissions["users.admin"]
+                })
+              }}
+            </div>
+          </div>
+
+          <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
+            <div class="ons-grid__col">
+            {{
+                onsButton({
+                    "id": "save-changes",
+                    "text": "Save changes",
+                    "submitType": "timer"
+                })
+            }}
+            </div>
+            <div class="ons-grid__col ons-u-ml-m">
+                <a href=" {{ url_for('admin_bp.manage_user_accounts') }}">Cancel</a>
+            </div>
+          </div>
+          <a href=" {{ url_for('admin_bp.get_delete_uaa_user', user_id=user_id) }}">Delete user account</a>
```

Comment:
Can you explain what you mean more sorry?  I don't understand.

Reply:
I always like to (try) to keep the html to presentation with the logic in the backend. In this example you are passing the user_id  https://github.com/ONSdigital/response-operations-ui/blob/7aca5ffa1bd89be4a5b1465aa84da37ffe80274e/response_operations_ui/views/admin/manage_user_accounts.py#L95 just to be used in one place, which then gets passed into another function (jinga by the looks of it). Instead of doing this the url generation could just be done in def manage_account and then pass a clean url var removing the need for {{ url_for('admin_bp.manage_user_accounts') }} in the template

Reply:
I guess we could.. but I'm not really convinced.  What I'm doing here is an incredibly common pattern throughout response ops and changing it to what you're suggesting doesn't really add much I think.

Reply:
No problem, it's only a suggestion